src/main/resources/sample_fields.json

```diff
@@ -0,0 +1,128 @@
+[
+    {
+        "name": "a",
+        "type": "STRING",
+        "description": "This is a string field"
+    },
+    {
+        "name": "b",
+        "type": "INTEGER",
+        "description": "This is an integer field"
+    },
+    {
+        "name": "c",
+        "type": "FLOAT",
+        "description": "This is a float field"
+    },
+    {
+        "name": "d",
+        "type": "DOUBLE",
+        "description": "This is a double field"
+    },
+    {
+        "name": "e",
+        "type": "LONG",
+        "description": "This is a long field"
+    },
+    {
+        "name": "f",
+        "type": "BOOLEAN",
+        "description": "This is a boolean field"
+    },
+    {
+        "name": "g",
+        "type": "STRING_LIST",
+        "description": "This is a list of strings"
+    },
+    {
+        "name": "h",
+        "type": "DOUBLE_MAP",
+        "description": "This is a map of doubles"
+    },
+    {
+        "name": "i",
+        "type": "FLOAT_MAP_MAP",
+        "description": "This is a map of map of floats"
+    },
+    {
+        "name": "j",
+        "type": "INTEGER_MAP_LIST",
+        "description": "This is a list of map of floats"
```

Comment:
excuse me this is a list of map of integers

Reply:
I am allowed to lie in descriptions